pypit/artrace.py

```diff
@@ -656,11 +610,11 @@ def trace_slits(slf, mstrace, det, pcadesc="", maskBadRows=False):
             lcnt = 1
         else:
             msgs.error("Unable to trace any edges"+msgs.newline()+"try a different method to trace the order edges")
-    elif (rcnt == 0) and (lcnt == 1):
+    elif rcnt == 0:
```

Comment:
I forget why I added the (lcnt == 1)
but it was necessary for one of the instruments.

Although by necessary, that may mean there was a 'bug' of sorts
and this was dealing with it..